test/Build.centos7xz

```diff
@@ -19,9 +19,9 @@
 
 set -ex
 
-srcdir=$1
+#srcdir=$1  # unused
```

Comment:
What's the reasoning for leaving these variables commented out instead of removing them?

Reply:
To be consistent, i.e., always mention all three arguments in the same order. Commenting them out satisfies ShellCheck.